dbt_project.yml

```diff
@@ -30,6 +30,7 @@ vars:
     intercom__using_team: true
     intercom__company_history_pass_through_columns: []
     intercom__contact_history_pass_through_columns: []
+    intercom__conversation_history_pass_through_columns: []
```

Comment:
Just wanted to call out that these are actually not needed and can sometimes mess with the dbt project compilation. I would prefer we keep them in this release (I don't want to accidentally break something in a patch update), but just wanted to highlight for future builds.